### PR TITLE
Updated switch topic

### PR DESCRIPTION
**Pull request by YoannaIvanova**

### CODE REVIEW

File: en/components/switch.md [hunk 1]
```diff
@@ -37,63 +43,25 @@ The Checkbox provides **on** and off selection states with additional variants f
 
 ## Styling
 
-The Switch comes with styling flexibility, allowing control over the thumb and track colors. There is a fixed alpha value applied to the track to make it semi transparent.
+The Switch comes with styling flexibility, allowing control over the thumb and track colors, as well as change of the label text style color. There is a fixed alpha value applied to the track to make it semi transparent.
```
Comment (StefanIvanov) [inline en/components/switch.md:46]:
```suggestion
The Switch comes with styling flexibility, allowing control over the thumb and track colors, as well as changing the label text color. The alpha value of the track is fixed to achieve a semi-transparent look no matter what color is used and can only be adjusted globally through the `Symbol Master` that can be found on the 🧩 Components page in the Indigo.Design library.```

File: en/components/switch.md [hunk 2]
```diff
@@ -12,23 +12,29 @@ Use the Switch Component to allow the user make a selection, which most often ex
...
 The Switch can be used styled in **dark** and light variants to assure good readability and contrast for both lighter and darker backgrounds.
 
 <img class="responsive-img" src="../images/switch_dark.png" srcset="../images/switch_dark@2x.png 2x" />
 <img class="responsive-img" src="../images/switch_light.png" srcset="../images/switch_light@2x.png 2x" />
 
-## Label use
+## Layout Template
 
-The Switch is used **with** or without label. In order to hide the label, set its text value to a blank/space character and reduce the width of the component to e.g. 38px.
+The Switch supports a label position where the Switch label is placed after or before the Switch element.
```
Comment (StefanIvanov) [inline en/components/switch.md:30]:
```suggestion
The Switch supports a label position where the Switch label is placed before or after the Switch element.
```

**Review by StefanIvanov — CHANGES_REQUESTED** [review]

**Review by StefanIvanov — APPROVED** [review]

File: en/components/switch.md [hunk 3]
```diff
@@ -37,65 +43,25 @@ The Checkbox provides **on** and off selection states with additional variants f
 
 ## Styling
 
-The Switch comes with styling flexibility, allowing control over the thumb and track colors. There is a fixed alpha value applied to the track to make it semi transparent.
+The Switch comes with styling flexibility, allowing control over the thumb and track colors, as well as changing the label text color. The alpha value of the track is fixed to achieve a semi-transparent look no matter what color is used and can only be adjusted globally through the `Symbol Master` that can be found on the 🧩 Components page in the Indigo.Design library.```
```
Comment (sdimchevski) [inline en/components/switch.md:46]:
```suggestion
The Switch comes with styling flexibility, allowing control over the thumb and track colors, as well as changing the label text color. The alpha value of the track is fixed to achieve a semi-transparent look no matter what color is used and can only be adjusted globally through the `Symbol Master` that can be found on the 🧩 Components page in the Indigo.Design library.
```

**Review by sdimchevski — CHANGES_REQUESTED** [review]
Please remove ``` after library.

**Review by sdimchevski — APPROVED** [review]

**Pull request merged**